cygnus-ngsi/test/performance/README.md

```diff
@@ -37,20 +37,21 @@ nohup sh startAgent.sh --udp-port 0 --tcp-port 4444 &
 ```
   >**Properties**:
 ```
-* RUNTIME      - time of test duration (60 Sec by default)
-* THREADS      - threads number (1 by default)
-* RAMPUP       -  ramp up of threads (0 by default)
-* HOST         - IP or hostname main node(in case of clusters is Nginx)  (127.0.0.1 by default)
-* PORT         - port used by cygnus (5050 by default)
-* HOST_NODE_1  - IP or hostname of Node 1  (127.0.0.1 by default, if the property is not appends it is ignored)
-* HOST_NODE_2  - IP or hostname of Node 2  (127.0.0.1 by default, if the property is not appends it is ignored)
-* TESTNAME    - sink name (channel to test)
-* ORGANIZATION - organization name (HA by default)
+* RUNTIME       - time of test duration (1 second by default)
+* THREADS       - threads number (1 by default)
+* RAMPUP        - ramp up of threads (0 by default)
+* HOST          - IP or hostname main node (in case of clusters is Nginx)  (127.0.0.1 by default)
+* PORT          - port used by cygnus (5050 by default)
+* HOST_NODE_1   - IP or hostname of Node 1  (127.0.0.1 by default, if the property is not appends it is ignored)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1230/commits/9a0be7a99267a967204c5b8ec350595631fe8bd0